petstore-api-mssql/README.md

```diff
@@ -0,0 +1,51 @@
+# petstore-api-mssql (APICHAMP Sample)
+
+| Sample                                 | Endpoints | Datasources | Authentication | Infos                                                                           |
+|----------------------------------------|-----------|-------------|----------------|---------------------------------------------------------------------------------|
+| petstore-api-mssql                     | 5         | MSSQL       | -              | Petstore (Get, Save) with multiple SQL table relations in MSSQL                 |
+
+## Documentation
+Find the full documentation under: [docs.apichamp.com](https://docs.apichamp.com)
+
+## Requirements:
+- [Docker Client](https://docs.docker.com/get-started/overview/)
+
+> **You must accept the licence terms of MSSQL**: Accept the ACCEPT_EULA in your docker-compose.yaml file.
```

Comment:
Is grey and easy to read over, but the info is sufficient to understand what needs to be done. The code part underneath is more prominent

Reply:
Added warning sign.